src/EthereumNamingService.ts

```diff
@@ -27,11 +27,11 @@ export abstract class EthereumNamingService extends NamingService {
   };
 
   static readonly UrlMap: BlockhanNetworkUrlMap = {
-    mainnet: 'https://mainnet.infura.io',
-    ropsten: 'https://ropsten.infura.io',
-    kovan: 'https://kovan.infura.io',
-    rinkeby: 'https://rinkeby.infura.io',
-    goerli: 'https://goerli.infura.io',
+    mainnet: 'https://main-rpc.linkpool.io',
+    ropsten: 'https://ropsten-rpc.linkpool.io',
+    rinkeby: '', 
+    goerli: '', 
+    kovan: '', 
```

Comment:
It seems to be a breaking change for users that used infura with a rinkeby, goerli or kovan testnet

Reply:
The problem is that since infura start requiring the projectid it wasn't really working for them by default as well. The default URL that was there didn't contain any projectId. They would just get NamingServiceDown error because of that. With the current setup since the URL is empty '' they should receive a configurationError saying that provider URL is not configured. 

Users from those testnets should have been connecting their providers by themself even by now, so it shouldn't affect them at all.

Reply:
It was working for users who provided a PROJECT_ID though. With this change it won't anymore

Reply:
I don't think I really get it. If they were using infura factory they give project id and network and it constructs the correct URL. 
If they were using default settings like 
```
new Resolution( { blockchain: { ens: { network: "rinkeby" } } ) 
```
they have to provide the URL by themself. With the previous code, it would substitute the URL without project id and it still won't work.

So people who were using  those network had to use the manual configuration before like
```
new Resolution(  { blockchain: { ens: { network: "rinkeby", url: "https://rinkeby.infura.io/v3/<Project id> } }  )
```

With provider factories, this is not needed since provider took the configuration for testnets and such. 

Did I miss some other methods?

Reply:
Ah, there was no way of passing PROJECT_ID with `Blockchain` parameters except for giving the full url... Got it. Sorry, my bad then